instrumentation/racecar/lib/opentelemetry/instrumentation/racecar/process_message_subscriber.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module OpenTelemetry
+  module Instrumentation
+    # This class contains the ASN subsciber that instruments message processing
+    class ProcessMessageSubscriber
+      def tracer
+        Racecar::Instrumentation.instance.tracer
+      end
+
+      def start(_name, _id, payload)
```

Comment:
I believe we're satisfying the `ActiveSupport::Subscriber` [interface](https://api.rubyonrails.org/classes/ActiveSupport/Subscriber.html) here. If so, can we explicitly say so in a comment? IDK if it makes sense to subclass the `Subscriber` class, but that would perhaps be another way of achieving this.